### PR TITLE
Add timer for marking glucose as stale

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-982

### CODE REVIEW

File: Loop/View Controllers/StatusTableViewController.swift [hunk 1]
```diff
@@ -619,6 +619,10 @@ final class StatusTableViewController: ChartsTableViewController {
 
         let hudIsVisible = self.shouldShowHUD
         let statusIsVisible = self.shouldShowStatus
+        
+        if hudIsVisible != hudView?.glucoseHUD.isVisible {
```
Comment (darinkrauss) [inline Loop/View Controllers/StatusTableViewController.swift]: [nit] Why not just set the `isVisible` property value and let the property's `didSet` determine if it changed and take appropriate action?

**Review by darinkrauss — APPROVED** [review]
Just one nit. Otherwise, LGTM! :shipit:

**Review by nhamming — APPROVED** [review]
LGTM

**Review by darinkrauss — APPROVED** [review]
LGTM! 🚀

**Pull request merged**